### PR TITLE
tutorial: add doc for store limit

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add doc for store limit.

> **Note**: Each branch is slightly different from other branches. Please modify them after cherry-picking.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3353, https://github.com/pingcap/docs-cn/pull/3831 (for master only)
- Other reference link(s):

### CODE REVIEW

File: configure-store-limit.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+### Persist store limit modification
+
+Because the store limit is a mapping in the memory, the above modification is reset after the leader is switched or 
+PD is restarted. If you want to persist the modification, run the following command:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+config set store-balance-rate 20    // All stores can at most add 20 learners/peers or delete 20 peers per minute
```
Comment (lilin90) [inline configure-store-limit.md:79]:
```suggestion
config set store-balance-rate 20    // All stores can at most add 20 learners/peers or delete 20 peers per minute.
```

File: configure-store-limit.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+title: Store Limit
+summary: Learn the feature of Store Limit.
+category: how-to
+---
+
+# Store Limit
+
+Store Limit is a feature of PD, introduced in TiDB 4.0. It is designed to control the scheduling speed in a finer manner for better performance in different scenarios.
```
[the remark this answers is not in the document]
Comment (ran-huang) [reply configure-store-limit.md:9]: Just confirmed with the author. This is a 3.0 feature, but we don't have the relevant docs for v3.0 and v3.1 yet, which are slightly different from the dev version. 
He'll create a PR to add them.

Comment (ran-huang) [inline configure-store-limit.md:9]:
```suggestion
Store Limit is a feature of PD, introduced in TiDB 3.0. It is designed to control the scheduling speed in a finer manner for better performance in different scenarios.
```

**Comment by ran-huang** [comment]
@lilin90 PTAL again. Thanks~

**Comment by ran-huang** [comment]
The changes in https://github.com/pingcap/docs-cn/pull/3831 has been aligned in this commit https://github.com/pingcap/docs/pull/2947/commits/2d8c671b24b7e7437b996d252994a5a93cada96b.

File: configure-store-limit.md [hunk 3]
```diff
@@ -51,9 +51,9 @@ To set the speed limit for all stores, run the following commands:
 {{< copyable "shell-regular" >}}
 
 ```bash
-store limit all 5                   // All stores can at most add 5 learns/peers per minute (if a specific type is not set, this command sets the speed of adding learners/peers).
-store limit all 5 region-add        // All stores can at most add 5 learns/peers per minute.
-store limit all 5 region-remove     // All stores can at most delete 5 learns/peers per minute.
+store limit all 5                   // All stores can at most add or delete 5 peers per minute.
```
Comment (rleungx) [inline configure-store-limit.md:54]: ditto

File: configure-store-limit.md [hunk 4]
```diff
@@ -39,9 +39,9 @@ To view the limit setting of the current store, run the following commands:
 {{< copyable "shell-regular" >}}
 
 ```bash
-store limit                         // Shows the speed limit of adding learners/peers in all stores (if a specific type is not set, this command shows the speed of adding learners/peers).
-store limit region-add              // Shows the speed limit of adding learners/peers in all stores.
-store limit region-remove           // Shows the speed limit of deleting learners/peers in all stores. 
+store limit                         // Shows the speed limit of adding or deleting peers in all stores.
```
Comment (rleungx) [inline configure-store-limit.md:42]:
```suggestion
store limit                         // Shows the speed limit of adding and deleting peers in all stores.
```

File: configure-store-limit.md [hunk 5]
```diff
@@ -63,18 +63,7 @@ To set the speed limit for a single store, run the following commands:
...
-### Persist store limit modification
-
-Because the store limit is a mapping in the memory, the above modification is reset after the leader is switched or 
-PD is restarted. If you want to persist the modification, run the following command:
-
-{{< copyable "shell-regular" >}}
-
-```bash
-config set store-balance-rate 20    // All stores can at most add 20 learners/peers or delete 20 peers per minute.
+store limit 1 5                     // store 1 can at most add or delete 5 peers per minute.
```
Comment (rleungx) [inline configure-store-limit.md:66]: ditto

File: configure-store-limit.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+### Set limit for all stores
+
+To set the speed limit for all stores, run the following commands:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+store limit all 5                   // All stores can at most add or delete 5 peers per minute.
```
Comment (ran-huang) [inline configure-store-limit.md:54]:
```suggestion
store limit all 5                   // All stores can at most add and delete 5 peers per minute.
```

File: configure-store-limit.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+### Set limit for a single store
+
+To set the speed limit for a single store, run the following commands:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+store limit 1 5                     // store 1 can at most add or delete 5 peers per minute.
```
Comment (ran-huang) [inline configure-store-limit.md:66]:
```suggestion
store limit 1 5                     // store 1 can at most add and delete 5 peers per minute.
```

File: configure-store-limit.md (hunk 5 above)
Comment (ran-huang) [reply configure-store-limit.md]: Updated. Thanks!

**Comment by ran-huang** [comment]
@lilin90 @rleungx All comments addressed. PTAL again. Thanks~

**Pull request merged**